front/lib/api/assistant/conversation.ts

```diff
@@ -1768,22 +1769,52 @@ export async function postNewContentFragment(
     return cfBlobRes;
   }
 
+  const supersededContentFragmentId = cf.supersededContentFragmentId;
+  // If the request is superseding an existing content fragment, we need to validate
+  // that it exists and is part of the conversation.
+  if (supersededContentFragmentId) {
+    let found = false;
+    for (const versions of conversation.content) {
```

Comment:
`nit` we could use `conversation.content.some()` to avoid the for loop.

Reply:
Indeed thanks